cocos/rendering/render-instanced-queue.ts

```diff
@@ -88,7 +88,7 @@ export class RenderInstancedQueue {
      */
     public recordCommandBuffer (device: Device, renderPass: RenderPass, cmdBuff: CommandBuffer,
         descriptorSet: DescriptorSet | null = null, dynamicOffsets?: Readonly<number[]>) {
-        const it = this._renderQueue.length === 0 ? this.queue.values() : this._renderQueue.values();
+        const it = this._renderQueue.length === 0 ? this.queue.values() : this._renderQueue[Symbol.iterator]();
```

Comment:
What's the difference between `this._renderQueue.values()` and `this._renderQueue[Symbol.iterator]()` ?

Reply:
_renderQueue is a JS Array. In my mini game, if using the former, there will be a persistent error message "TypeError: this._renderQueue.values() is not a function" on Android phone, but there is no such problem on iOS phones. If using the latter, there are no issues on either platform.

Reply:
Got it. May I ask which mini-game platform did you test on?

Reply:
If this is an issue, all places where using `.values()` needs to be modified too.

Reply:
Wechat on XiaoMI 8

Reply:
The scope of impact should not be large. As in the example above, this.queue is a JS Set and it is normal to use values to get an iterator. However, this._renderQueue is an JS Array and using values is a bit uncommon.